techempower/scripts/tfb-qrh-deploy-openshift.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
```

Comment:
@kusumachalasani - Can you include a README with details on how to run this benchmark